### PR TITLE
updated with modal on downloading  bpmn

### PR DESCRIPTION
**Pull request by abilpraju-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-3759
https://aottech.atlassian.net/browse/FWF-3808
https://jam.dev/c/95324e5e-1676-4d1d-9f47-adf18bc9151c
# Changes

# Screenshots (if applicable)

# Notes

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

File: forms-flow-web/src/components/Modeler/Editors/BpmnEditor/index.js [hunk 1]
```diff
@@ -82,6 +98,56 @@ const BpmnEditor = forwardRef(({ bpmnXml, setLintErrors }, ref) => {
     handleImport: (bpmnXml) => handleImport(bpmnXml),
   }));
 
+  
+ 
+  const validateProcess = async (xml) => {
+    // If the BPMN Linting is active then check for linting errors, else check for Camunda API errors
+    // Check for linting errors in the modeler view
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Modeler/Editors/BpmnEditor/index.js:105]: don't write any function inside this component please make sure the bpmn editor is reusable component only pass the xml and error setting function, and you can pass the ref and take the xml data by calling getBpmnModler function using this ref @abilpraju-aot  so please rework this part

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/components/Modals/ActionModal.js [hunk 2]
```diff
@@ -115,8 +117,18 @@ const ActionModal = React.memo(({ newActionModal, onClose, CategoryType, onActio
                                     className=""
                                     dataTestid="export-workflow-button"
                                     ariaLabel="Export Workflow"
+                                    onClick={() => handleAction("EXPORT")}
+                                />
+                           <CustomButton
+                                    variant="secondary"
+                                    size="sm"
+                                    label="Delete"
+                                    icon={<CloseIcon color="#253DF4" />}
```
Comment (arun-s-aot) [inline forms-flow-web/src/components/Modals/ActionModal.js:126]: in the entire file , wherever we have added color="#253DF4" , please replace with css variable 
var(--ff-base-600) . It is present in entire page. 
Also if we dont have classNames to add , we can remove it from all the CustomButtons in this page 

@abilpraju-aot

**Review by arun-s-aot — APPROVED** [review]

**Comment by arun-s-aot** [comment]
@abilpraju-aot Please remove the className="" attribute also as per previous comment

**Review by auslin-aot — APPROVED** [review]

File: forms-flow-web/src/apiManager/services/processServices.js [hunk 3]
```diff
@@ -470,11 +471,10 @@ export const getProcessActivities = (process_instance_id, ...rest) => {
...
   process_key,
   tenant_key = null,
-  isDmn = false,
   ...rest
 ) => {
-  const api = isDmn ? API.DMN_XML : API.PROCESSES_XML;
-
+  const api = API.GET_PROCESS_XML;
+ console.log("api",api,process_key);
   let url = replaceUrl(api, "<process_key>", process_key);
```
Comment (auslin-aot) [inline forms-flow-web/src/apiManager/services/processServices.js:478]: please remove console

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**